doc/whats-new.rst

```diff
@@ -70,6 +70,13 @@ Breaking changes
 Deprecations
 ~~~~~~~~~~~~
 
+- Removed the deprecated ``dim`` kwarg to :py:func:`DataArray.integrate` (:pull:`5630`)
+- Removed the deprecated ``keep_attrs`` kwarg to :py:func:`DataArray.rolling` (:pull:`5630`)
+- Removed the deprecated ``keep_attrs`` kwarg to :py:func:`DataArray.coarsen` (:pull:`5630`)
+- Completed deprecation of passing an ``xarray.DataArray`` to :py:func:`Variable` - will now raise a ``TypeError`` (:pull:`5630`)
+- Removed the deprecated return value of :py:func:`Dataset.update` (:pull:`5630`)
+- Removed the deprecated lock kwarg to :py:func:`DataArray.open_dataset` (:pull:`5630`)
```

Comment:
I didn't do that one, as I read the deprecation message as "the deprecation will be completed in the version after 0.19". Should we remove this?
```suggestion
```

Reply:
Ah okay - yes let's remove that